Model/Cron.php

```diff
@@ -449,7 +451,8 @@ public function execute()
                         $this->_adyenLogger->addAdyenNotificationCronjob('Going to cancel the order');
 
                         // if payment is API check, check if API result pspreference is the same as reference
-                        if ($this->_eventCode == NOTIFICATION::AUTHORISATION && $this->_getPaymentMethodType() == 'api') {
+                        if ($this->_eventCode == NOTIFICATION::AUTHORISATION && $this->_getPaymentMethodType(
+                            ) == 'api') {
```

Comment:
Maybe this is better, unless it is rejected by the marketplace
```suggestion
                        if ($this->_eventCode == NOTIFICATION::AUTHORISATION 
                        && $this->_getPaymentMethodType() == 'api') {
```